### DISCUSSION TITLE
Backups are not listing the Backup section of the Longhorn Dashboard

### DISCUSSION THREAD

**Discussion by reddybhavaniprasad** [longhorn/longhorn#7813] [Q&A]
I am trying to take longhorn dump of some 4 volumes running **v1.3.0** and planned to restore them in the longhorn running on **v1.4.3**. The setup came up properly and after updating the backup target in the General Setting section, I am facing an issue with instance manager pod when clicked on Backup option as follows,
```
failed to get engine client proxy: instance-manager-e-439cfb0f4e8a19fc808b4cb6cf923b72 instance manager is in stopped, not running state
```
![image](https://github.com/longhorn/longhorn/assets/31069365/ad8f53a5-5f87-4473-bced-4a6e5b47d889)

strangely this pod is not at all present on the **longhorn-system** namespace at all. The issue is persisting even after deploying longhorn for several number of times.
Actually the same procedure worked fine in another clusters but am facing this issue only in a particular cluster.

Please share me if I can try something to overcome this issue and successfully restore the longhorn dump.

**Comment by c3y1huang** [comment]
Have you checked the longhorn-manager pod logs for any errors? Are you able to share the support bundle for analysis?
